contracts/strategy/yearn/YearnStrategy.sol

```diff
@@ -89,15 +89,20 @@ contract YearnStrategy is IStrategy, AccessControl, CustomErrors {
         return _getShares() > 0;
     }
 
-    /// @inheritdoc IStrategy
+    /**
```

Comment:
Why this change? This should be in IStrategy

Reply:
My bad, [fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/139/commits/35122021e003fa1dc07af950485cf15406b0a436)